features/pageobjects/dynamic.loading.page.js

```diff
@@ -0,0 +1,37 @@
+const { $ } = require('@wdio/globals')
+const Page = require('./page');
+
+
+class DynamicLoadingPage extends Page {
+    
+    get Example2Link () {
+        return $('/html/body/div[2]/div/div/a[2]');

```

Comment:
And what if you will need example 1 link at some point in the future? Instead of hardcoding the locator, make the locator parametrized like exampleLink(number) { ... } 
Also this kind of locator is very unstable, there's a way to improve it